package.json

```diff
@@ -1025,7 +1025,7 @@
     }
   },
   "engines": {
-    "vscode": "^1.26.0"
+    "vscode": "^1.33.0"
```

Comment:
Hmm Stephen had a comment related to this and then deleted it...but I still think this should be 1.31.0 (at least for this PR) - the minimum needed for `vscode.env.clipboard`